mantidproject/mantid
# Disable network access on startup for mpi installs

**Description of work.**

Disables updating instruments, reporting usage and checking for updates on startup for MPI-enabled installs.

**To test:**

Build with MPI, install and check that reporting & updating is disabled.

*No issue*

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
